vector/src/main/res/layout/fragment_ftue_auth_use_case.xml

```diff
@@ -0,0 +1,167 @@
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:background="@drawable/bg_login_server_selector"
+        android:drawablePadding="16dp"
+        android:gravity="center_vertical"
+        android:padding="16dp"
+        android:text="@string/ftue_auth_use_case_option_one"
+        android:textColor="?vctr_content_primary"
+        app:drawableStartCompat="@drawable/ic_friends_and_family"
+        app:layout_constraintBottom_toTopOf="@id/useCaseOptionTwo"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/titleContentSpacing" />
+
+    <TextView
+        android:id="@+id/useCaseOptionTwo"
+        style="@style/Widget.Vector.TextView.Subtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:background="@drawable/bg_login_server_selector"
+        android:drawablePadding="16dp"
+        android:gravity="center_vertical"
+        android:padding="16dp"
+        android:text="@string/ftue_auth_use_case_option_two"
+        android:textColor="?vctr_content_primary"
+        app:drawableStartCompat="@drawable/ic_teams"
+        app:layout_constraintBottom_toTopOf="@id/useCaseOptionThree"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/useCaseOptionOne" />
+
+    <TextView
+        android:id="@+id/useCaseOptionThree"
+        style="@style/Widget.Vector.TextView.Subtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:background="@drawable/bg_login_server_selector"
+        android:drawablePadding="16dp"
+        android:gravity="center_vertical"
+        android:padding="16dp"
+        android:text="@string/ftue_auth_use_case_option_three"
+        android:textColor="?vctr_content_primary"
+        app:drawableStartCompat="@drawable/ic_communities"
+        app:layout_constraintBottom_toTopOf="@id/useCaseSkip"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/useCaseOptionTwo" />
+
+    <TextView
+        android:id="@+id/useCaseSkip"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:clickable="true"
+        app:layout_constraintBottom_toTopOf="@id/contentFooterSpacing"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/useCaseOptionThree" />
```

Comment:
maybe add a `tools:text` attribute for a better preview of the layout

Reply:
1f876b9088363817f8a182bbf007b5b3b94787a8